content/francais/python-turtle/the-basics.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Les bases - Dessiner une ligne colorée"
+date: 2024-09-17T13:24:17-07:00
+weight: 2
+draft: false
+---
+
+Avant qu'Alex ne puisse apprendre à créer une ruche, il doit d'abord apprendre à dessiner une ligne colorée. Nous ajouterons du code à la fenêtre Trinket ci-dessous, et nous le ferons en deux parties.
+
+Tout d'abord, nous devons dire à la tortue quelle couleur nous voulons que la ligne soit. Pour cela, tapons :
+
+``` python
+turtle.color("orange")
+```
+
+Essayons de comprendre cette ligne de code plus en détail.
+
+{{% notice tip %}}
+
+## Chaînes de caractères
+
+Tout ce que vous voyez entre **guillemets** s'appelle une **chaîne de caractères**. Une chaîne est simplement une séquence de caractères (qu'il s'agisse de lettres, de chiffres ou de symboles).
```

Comment:
**string** should not be translated as is a type/reserved word. 
change "**chaîne de caractères**" for **string**

Reply:
Same goes for further in the same line:
" Une chaîne est simplement une séquence de caractères " change to
 "Une **string** est simplement une...."